app/views/games/show.html.erb

```diff
@@ -0,0 +1,93 @@
+<div class="game">
+  <h1>Have Fun</h1>
+  <br />
+  <table id="gameboard">
+    <tbody>
+      <tr class="y-pos-0">
+        <td class="x-pos-7"></td>
+        <td class="x-pos-6"></td>
+        <td class="x-pos-5"></td>
+        <td class="x-pos-4"></td>
+        <td class="x-pos-3"></td>
+        <td class="x-pos-2"></td>
+        <td class="x-pos-1"></td>
+        <td class="x-pos-0"></td>
+      </tr>
+      <tr class="y-pos-1">
+        <td class="x-pos-7"></td>
+        <td class="x-pos-6"></td>
+        <td class="x-pos-5"></td>
+        <td class="x-pos-4"></td>
+        <td class="x-pos-3"></td>
+        <td class="x-pos-2"></td>
+        <td class="x-pos-1"></td>
+        <td class="x-pos-0"></td>
+      </tr>
+      <tr class="y-pos-2">
+        <td class="x-pos-7"></td>
+        <td class="x-pos-6"></td>
+        <td class="x-pos-5"></td>
+        <td class="x-pos-4"></td>
+        <td class="x-pos-3"></td>
+        <td class="x-pos-2"></td>
+        <td class="x-pos-1"></td>
+        <td class="x-pos-0"></td>
+      </tr>
+      <tr class="y-pos-3">
+        <td class="x-pos-7"></td>
+        <td class="x-pos-6"></td>
+        <td class="x-pos-5"></td>
+        <td class="x-pos-4"></td>
+        <td class="x-pos-3"></td>
+        <td class="x-pos-2"></td>
+        <td class="x-pos-1"></td>
+        <td class="x-pos-0"></td>
+      </tr>
+      <tr class="y-pos-4">
+        <td class="x-pos-7"></td>
+        <td class="x-pos-6"></td>
+        <td class="x-pos-5"></td>
+        <td class="x-pos-4"></td>
+        <td class="x-pos-3"></td>
+        <td class="x-pos-2"></td>
+        <td class="x-pos-1"></td>
+        <td class="x-pos-0"></td>
+      </tr>
+      <tr class="y-pos-5">
+        <td class="x-pos-7"></td>
+        <td class="x-pos-6"></td>
+        <td class="x-pos-5"></td>
+        <td class="x-pos-4"></td>
+        <td class="x-pos-3"></td>
+        <td class="x-pos-2"></td>
+        <td class="x-pos-1"></td>
+        <td class="x-pos-0"></td>
+      </tr>
+      <tr class="y-pos-6">
+        <td class="x-pos-7"></td>
+        <td class="x-pos-6"></td>
+        <td class="x-pos-5"></td>
+        <td class="x-pos-4"></td>
+        <td class="x-pos-3"></td>
+        <td class="x-pos-2"></td>
+        <td class="x-pos-1"></td>
+        <td class="x-pos-0"></td>
+      </tr>
+      <tr class="y-pos-7">
+        <td class="x-pos-7"></td>
+        <td class="x-pos-6"></td>
+        <td class="x-pos-5"></td>
+        <td class="x-pos-4"></td>
+        <td class="x-pos-3"></td>
+        <td class="x-pos-2"></td>
+        <td class="x-pos-1"></td>
+        <td class="x-pos-0"></td>
+      </tr>
+    </tbody>
+  </table>
+  <!-- <br /> -->
+  <div class="game-nav-item">

```

Comment:
Should we add something here to make sure this option to 'Sign In' only appears for users who are not currently signed in? As a signed-in user, the option to sign in still appears, which could be confusing.
Something like this:
 <% if user_signed_in? %>
            <li class = "game-nav-item">
              <%= link_to 'Sign Out', destroy_user_session_path, method: :delete, class: 'nav-link' %>
            </li>
          <% else %>
            <li class ="game-nav-item">
              <%= link_to 'Sign In', new_user_session_path, class: 'nav-link' %>
            </li>
            <li class ="nav-item">
              <%= link_to 'Sign Up', new_user_registration_path, class: 'nav-link' %>
            </li>
          <% end %>

Reply:
We could also remove this part completely-when you view the page as a logged in user, the signout option remains in the top nav bar, and as a new user both the sign in/sign up options appear in the top nav bar as well.

Reply:
@juanlavieri